java/src/test/java/org/hyperledger/fabric/client/FileCheckpointerTest.java

```diff
@@ -0,0 +1,55 @@
+package org.hyperledger.fabric.client;
+
+
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class FileCheckpointerTest extends CommonCheckpointerTest {
+    FileCheckpointer fileCheckpointer;
```

Comment:
Vsibility and we're already in a FileCheckpointerTest so the "file" part of the variable naming is kinda implied.
```suggestion
    private FileCheckpointer checkpointer;
```